### ISSUE TITLE
normalize() argument 2 must be str, not numpy.ndarray

### ISSUE THREAD

**Issue by githubsumanlm10** [pytorch/tutorials#1002]
`txt = pd.read_csv('/content/drive/My Drive/Bert-text/Musical_instruments_reviews.csv')
txt.head()
txt = txt.filter(['reviewText', 'score'])
train, test = train_test_split(txt, test_size=0.2, random_state=42)
x_train = np.array(train.reviewText)
y_train = np.array(train.score)

x_test = np.array(test.reviewText)
y_test = np.array(test.score)
(x_train, y_train), (x_test, y_test), preproc = text.texts_from_array(x_train=x_train, y_train=y_train,
                                                                      x_test = x_test, y_test=y_test, 
                                                                      class_names=['bad', 'neutral', 'good'],
                                                                      preprocess_mode='bert', maxlen=150)`
